src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,182 @@
+        loadFile();
+
+        if (kits.remove(name) == null) throw NOT_FOUND_EXCEPTION.create(name);
+        saveFile();
+        sendFeedback("commands.ckit.delete.success", name);
+        return 1;
+    }
+
+    private static int edit(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.put(name, client.player.inventory) == null) {
+            saveFile();
+            sendFeedback("commands.ckit.edit.notFoundSoCreatedNew", name);
+        } else {
+            saveFile();
+            sendFeedback("commands.ckit.edit.success", name);
+        }
+        return 1;
+    }
+
+    private static int kit(ServerCommandSource source, String name, boolean override) throws CommandSyntaxException {
+        if (!client.player.isCreative() || !client.player.abilities.creativeMode) throw NOT_CREATIVE_EXCEPTION.create();
+
+        loadFile();
+
+        PlayerInventory kit = kits.get(name);
+        if (kit == null) throw NOT_FOUND_EXCEPTION.create(name);
+
+        for (int i = 0; i < kit.main.size(); i++) {
+            if (kit.main.get(i) == ItemStack.EMPTY && !override) continue;
+            client.player.inventory.main.set(i, kit.main.get(i));
+        }
+        for (int i = 0; i < kit.armor.size(); i++) {
+            if (kit.armor.get(i) == ItemStack.EMPTY && !override) continue;
+            client.player.inventory.armor.set(i, kit.armor.get(i));
+        }
+        for (int i = 0; i < kit.offHand.size(); i++) {
+            if (kit.offHand.get(i) == ItemStack.EMPTY && !override) continue;
+            client.player.inventory.offHand.set(i, kit.offHand.get(i));
+        }
+
+        List<Slot> slots = client.player.playerScreenHandler.slots;
+        for (int i = 0; i < slots.size(); i++) {
+            if (slots.get(i).inventory == client.player.inventory) {
+                client.interactionManager.clickCreativeStack(kit.getStack(((ISlot) slots.get(i)).getIndex()), i);
+            }
+        }
+
+        client.player.playerScreenHandler.sendContentUpdates();
+        sendFeedback("commands.ckit.kit.success", name);
+        return 1;
+    }
+
+    private static int kits(ServerCommandSource source) throws CommandSyntaxException {
+        loadFile();
+
+        sendFeedback("Available kits: " + String.join(", ", kits.keySet()));
+        return 1;
+    }
+
+    private static void saveFile() throws CommandSyntaxException {
+        try {
+            CompoundTag compoundTag = new CompoundTag();
+
+            for (Map.Entry<String, PlayerInventory> entry : kits.entrySet()) {
+                compoundTag.put(entry.getKey(), entry.getValue().serialize(new ListTag()));
+            }
+
+            File file = File.createTempFile("kits", ".dat", path.toFile());
+            NbtIo.write(compoundTag, file);
+            File file2 = new File(path.toFile(), "kits.dat_old");
+            File file3 = new File(path.toFile(), "kits.dat");
```

Comment:
`file`, `file1` and `file2` could have more descriptive names. Like the idea of creating a backup though :+1: